### PR TITLE
Support collections in log and single value in add/remove

### PR DESCRIPTION
**Pull request by aniezurawski**

### CODE REVIEW

File: neptune/alpha/experiment.py [hunk 1]
```diff
@@ -133,6 +133,9 @@ def set_attribute(self, path: str, attribute: Attribute) -> Optional[Attribute]:
         with self._lock:
             return self._structure.set(parse_path(path), attribute)
 
+    def exists(self, path: str) -> bool:
+        return self.get_attribute(path) is not None
```
Comment (szymon-kuklewicz) [inline neptune/alpha/experiment.py:137]: Suggestion to define `__bool__` instead.

File: neptune/alpha/handler.py [hunk 2]
```diff
@@ -73,26 +75,39 @@ def save_files(self, value: Union[str, Iterable[str]], wait: bool = False) -> No
...
                     attr = ImageSeries(self._experiment, parse_path(self._path))
                 attr.log(value, step=step, timestamp=timestamp, wait=wait)
                 self._experiment.set_attribute(self._path, attr)
             else:
                 attr.log(value, step=step, timestamp=timestamp, wait=wait)
 
-    def add(self, values: Iterable[str], wait: bool = False) -> None:
-        verify_collection_type("values", values, str)
+    def add(self, values: Union[str, Iterable[str]], wait: bool = False) -> None:
+        verify_type("values", values, (str, Iterable))
```
Comment (szymon-kuklewicz) [inline neptune/alpha/handler.py:110]: verify_collection_type should be called for a collection

File: neptune/alpha/experiment.py (hunk 1 above)
Comment (aniezurawski) [reply neptune/alpha/experiment.py:137]: __bool__ don't take arguments and is less intuitive for users.

File: neptune/alpha/handler.py (hunk 2 above)
Comment (aniezurawski) [reply neptune/alpha/handler.py:110]: It is verified in attr.add()

**Review by szymon-kuklewicz — APPROVED** [review]

**Pull request merged**